.rgignore

```diff
@@ -0,0 +1 @@
+vendor
```

Comment:
what is this file for ?
should it be `.gitignore` ?

Reply:
[ripgrep ](https://github.com/BurntSushi/ripgrep) ignore file (who is able to use .gitignore files).

We don't use this tool let's delete this file.